test/method/testbader.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2020, the cclib development team
+#
+# This file is part of cclib (http://cclib.github.io) and is distributed under
+# the terms of the BSD 3-Clause License.
+
+"""Test the various population analyses (MPA, LPA, CSPA, Bickelhaupt) in cclib"""
+
+from __future__ import print_function
+
+import sys
+import os
+import logging
+import unittest
+
+import numpy
+
+from cclib.method import Bader, Volume
+from cclib.parser import Psi4
+from cclib.method.calculationmethod import MissingAttributeError
+
+from numpy.testing import assert_allclose
+
+sys.path.insert(1, "..")
```

Comment:
I'm actually not sure about this. I've noticed that this line is included in every test script in `method` so I assumed that this is needed for Travis. Can someone let me know what the intended function of these were?

Reply:
See 3026637b51b205ae9fcc32188d226c9d3da3a1c6.

The reason we manipulate `sys.path` at all (I think) is because the main `test` directory isn't in `PYTHONPATH`, and we need the relative `test_data` import to work.

We might be able to rip all of these out in the future, since now we're using `pytest` and not `unittest`, and I'd like to move the tests under the main `cclib` directory.

Reply:
Sounds about right. Sometimes it's hard to tell which legacy maintenance/test code can be removed. It's also somewhat unrewarding to work on it :)

Reply:
I've removed this line. I'll add an issue for other test functions. Thanks!